node/sc/sub_bridge_handler.go

```diff
@@ -357,37 +373,30 @@ func (sbh *SubBridgeHandler) broadcastServiceChainReceiptRequest() {
 	}
 }
 
-func (sbh *SubBridgeHandler) blockAnchoringManager(block *types.Block) {
-	startBlkNum := sbh.GetNextAnchoringBlockNumber()
-
-	var successCnt, cnt, blkNum uint64
-	latestBlkNum := block.Number().Uint64()
-
-	// TODO-Klaytn-Servicechain remove this code or define right confirmation block count.
-	// To consider a non-absolute finality consensus like PoA (Clique), this code anchors past blocks.
-	const confirmCnt = uint64(10)
-	if latestBlkNum > confirmCnt {
-		latestBlkNum = latestBlkNum - confirmCnt
-	} else {
-		return
+// updateTxCounts update txCounts to insert into anchoring tx. It skips first remnant txCounts.
+func (sbh *SubBridgeHandler) updateTxCounts(block *types.Block) {
```

Comment:
This function can be called by chainHead event.
This event doesn't deliver all blocks. So we need to check the blocks skipped in chainHead event.

Reply:
Fixed.